sklearn/datasets/_rcv1.py

```diff
@@ -136,8 +136,8 @@ def fetch_rcv1(
     Returns
     -------
     dataset : :class:`~sklearn.utils.Bunch`
-        Dictionary-like object. Returned only if `return_X_y` is False. Has
-        the following attributes.
+        Dictionary-like object. Returned only if `return_X_y` is False. 
```

Comment:
There is a failing test related to `black` because there is a trailing white space in this line.